docs/ops/movement/GatherElements_6.md

```diff
@@ -0,0 +1,128 @@
+## GatherElements <a name="GatherElements"></a> {#openvino_docs_ops_movement_GatherElements_6}
+
+**Versioned name**: *GatherElements-6*
+
+**Category**: Data movement operations
+
+**Short description**: *GatherElements* takes elements from the input `data` tensor at positions specified in the `indices` tensor.
+
+**Detailed description** *GatherElements* takes elements from the `data` tensor at positions specified in the `indices` tensor. 
+The `data` and `indices` tensors have the same rank `r >= 1`. Optional attribute `axis` determines 
+along which axis elements with indices specified in `indices` are taken. The `indices` tensor has the same shape as 
+the `data` tensor except for the `axis` dimension. Output consists of values (gathered from the `data` tensor) for each 
+element in the `indices` tensor and has the same shape as `indices`.
+
+For instance, in the 3D case (`r = 3`), the output is determined by the following equations:
+```
+  out[i][j][k] = data[indices[i][j][k]][j][k] if axis = 0
+  out[i][j][k] = data[i][indices[i][j][k]][k] if axis = 1
+  out[i][j][k] = data[i][j][indices[i][j][k]] if axis = 2
+```
+Example 1 with concrete values:
+```
+  data = [
+      [1, 2],
+      [3, 4],
+  ]
+  indices = [
+      [0, 1],
+      [0, 0],
+  ]
+  axis = 0
+  output = [
+      [
+        [1, 4],
+        [1, 2],
+      ],
+  ]
```

Comment:
Looks like output has shape [1, 2, 2] here, but it should be [2, 2].
The same comments for other examples.

Reply:
Thanks for noting that. Fixed.